### ISSUE TITLE
Long inventory uploads should show feedback while in progress

### ISSUE THREAD

**Issue by FuhuXia** [GSA/data.gov#2646]
## User Story

In order to have confidence that inventory is working correctly, an agency inventory manager should see feedback about the state of their upload in progress.

## Acceptance Criteria
[ACs should be clearly demoable/verifiable whenever possible. Try specifying them using [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications).]
- [ ] GIVEN I am an inventory user \
  AND I am logged into inventory.data.gov \
  AND I have a file that's 200MB in size \
  WHEN I upload the file \
  THEN I see a notice not to close the browser tab/window until the upload completes \
  AND I see a spinner or other indicator of activity until the upload is complete.

## Background
[Any helpful contextual notes or links to artifacts/evidence, if needed]

We have received support inquiries from users because uploads take a while and they get impatient... Users need to know that the lack of feedback is not an indicator of a problem, and that they should keep waiting.

## Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))
[comment]: # "Our SSP says 'The Data.gov team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.' so please don't remove this section without care."
[Any security concerns that might be implicated in the change. "None" is OK, just be explicit here!]
None.

## Sketch
[Notes or a checklist reflecting our understanding of the selected approach]Staging inventory is experiencing very slow bandwidth when upload or download dataset resources. Production should have same issue because user reports resource upload failure for a large (170MB) csv file.

**Comment by FuhuXia** [comment]
The **800KB/s** low speed only applies to **staging** Citrix VM environment. **Production** gives **4MB/s**,  still slower than **15MB/s** I got from **Sandbox**, but acceptable.  The resource upload failure reported by productions user appears to be cosmetic UI issue when dealing with large file upload.

**Comment by mogul** [comment]
@adborden indicated this may be a good issue for the ckan-multi team to take on.

**Comment by anuveyatsu** [comment]
@FuhuXia @adborden thanks for reporting this. I believe we have the info message while uploading a file (see screenshot below). We do not have a spinner or progress bar indicating how much of a file has been uploaded.

<img width="1244" alt="Screen Shot 2021-01-15 at 13 47 50" src="https://user-images.githubusercontent.com/17809581/104696353-5a516180-5738-11eb-83e0-4e6d25012fba.png">

**Comment by FuhuXia** [comment]
We need a more pronounced UI element to tell user not to navigate away at this point otherwise upload will fail.

**Comment by adborden** [comment]
Right, I think that given what we've seen from users, whatever messaging we have in place is not enough.
